crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -356,21 +356,19 @@ def upgrade_controller_nodes
         drbd_nodes = NodeObject.find("drbd_rsc:*")
         return upgrade_drbd_clusters unless drbd_nodes.empty?
 
-        founder = NodeObject.find(
-          "state:crowbar_upgrade AND pacemaker_founder:true"
-        ).first
+        founder = NodeObject.find("pacemaker_founder:true").first
         cluster_env = founder[:pacemaker][:config][:environment]
 
         non_founder = NodeObject.find(
-          "state:crowbar_upgrade AND pacemaker_founder:false AND " \
+          "pacemaker_founder:false AND " \
           "pacemaker_config_environment:#{cluster_env}"
         ).first
 
         return false unless upgrade_first_cluster_node founder.name, non_founder.name
 
         # upgrade the rest of nodes in the same cluster
         NodeObject.find(
-          "state:crowbar_upgrade AND pacemaker_config_environment:#{cluster_env}"
+          "pacemaker_config_environment:#{cluster_env}"
         ).each do |node|
           return false unless upgrade_next_cluster_node node.name
```

Comment:
Yes, there's a check for `node_api.upgraded?` at the start of `upgrade_next_cluster_node`